packages/design-system/src/components/Alert/Alert.test.tsx

```diff
@@ -4,19 +4,31 @@ import { UtagContainer } from '../analytics';
 import { setAlertSendsAnalytics } from '../flags';
 import { render, screen } from '@testing-library/react';
 
+import register from 'preact-custom-element';
+register(Alert, 'ds-alert');
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'ds-alert': AlertProps;
+    }
+  }
+}
+
 const defaultText = 'Ruhroh';
 
 function renderAlert(props: AlertProps = {}) {
   // eslint-disable-next-line react/no-children-prop
-  return render(<Alert children={defaultText} {...props} />);
+  // return render(<Alert children={defaultText} {...props} />);
+  return render(<ds-alert {...props}>{defaultText}</ds-alert>);
```

Comment:
to test the web component, we need to use the web component and not the Preact/React one.